Tools/WinMLRunner/README.md

```diff
@@ -35,6 +35,8 @@ Required command-Line arguments:
 -GPU : run model on default GPU
 -GPUHighPerformance : run model on GPU with highest performance
 -GPUMinPower : run model on GPU with the least power
+-GPUAdapterIndex <index number> : run model on GPU specified by its DXCore index. NOTE: Please only use this flag on DXCore supported machines.
```

Comment:
The adapter indexes are only guaranteed to be the same for the lifetime of the adapter list, and therefore we don't reliably know them ahead of time. I'd lean towards removing this option.

Reply:
But it is very convenient, also WinMLRunner prints out the available adapters.

Reply:
True - but there's no guarantee those indexes won't change from execution to execution.

Reply:
If it's not guaranteed from execution to execution. I would propose to remove it.

Reply:
Removed -GpuAdapterIndex.